xbmc/GUIInfoManager.cpp

```diff
@@ -5687,6 +5687,14 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///     @return The parental rating of the list item (PVR).
 ///     <p>
 ///   }
+///   \table_row3{   <b>`ListItem.CurrentItem`</b>,
+///                  \anchor ListItem_CurrentItem
+///                  _integer_,
```

Comment:
Type 'integer' does not match implementation, which returns a string. (GUIInfoManager.cpp line 9838). Do you only want to to display the value, then sting is correct. if you want to do greater, less etc math, then you should go with integer.

Reply:
I think display is enough.
I can't see a benefit of comparing them.

Reply:
Given the latest conversation I found it doesn't matter if its `_integer_`.
The sample with `ListItem.Year` is also a `_string_` and can be used in comparisons.
So I changed the documentation to align with the code.
`CurrentItem` is now `_string_`.